_includes/pbc/all/install-features/latest/install-the-order-amendment-feature.md

```diff
@@ -1892,6 +1859,545 @@ Verify that reordering is disabled for orders created from a quote request:
+        ];
+    }
+}
+```
+
+</details>
+
+#### Specify the payment methods that will enable the asynchronous order amendment checkout process flow.
+
+**src/Pyz/Zed/OrderAmendmentExample/OrderAmendmentExampleConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\SalesOrderAmendmentWidget;
+
+use Spryker\Zed\OrderAmendmentExample\OrderAmendmentExampleConfig as SprykerOrderAmendmentExampleConfig;
+
+class OrderAmendmentExampleConfig extends SprykerOrderAmendmentExampleConfig
+{
+    /**
+     * @var list<string>
+     */
+    protected const ASYNC_ORDER_AMENDMENT_PAYMENT_METHOD_NAMES = [
+        'dummyPaymentInvoice',
+    ];
+}
+```
+
+### Option 2: Enabling the async order amendment unconditionally
+
+To enable the asynchronous order amendment unconditionally, configure the default order amendment quote process flow which will be applied when the order amendment is started.
+
+#### Specify the order amendment quote process flow.
+
+**src/Pyz/Zed/SalesOrderAmendment/SalesOrderAmendmentConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\SalesOrderAmendment;
+
+use Generated\Shared\Transfer\QuoteProcessFlowTransfer;
+use Spryker\Zed\SalesOrderAmendment\SalesOrderAmendmentConfig as SprykerSalesOrderAmendmentConfig;
+
+class SalesOrderAmendmentConfig extends SprykerSalesOrderAmendmentConfig
+{
+    /**
+     * Specification:
+     * - Returns the default order amendment quote process flow.
+     *
+     * @api
+     *
+     * @return \Generated\Shared\Transfer\QuoteProcessFlowTransfer
+     */
+    public function getDefaultOrderAmendmentQuoteProcessFlow(): QuoteProcessFlowTransfer
+    {
+        return (new QuoteProcessFlowTransfer())->setName(SalesOrderAmendmentExtensionContextsInterface::CONTEXT_ORDER_AMENDMENT_ASYNC);
+    }
+}
+```
+
+#### Enable plugins for the asynchronous order amendment checkout process flow.
+
+Make sure all the needed plugins in modules dependency providers registered for the `order-amendment` checkout process flow are also registered for the `order-amendment-async` checkout process flow.
+So instead of dependency provider methods with `ForOrderAmendment` postfix use the methods with `ForOrderAmendmentAsync` postfix. Check the `Cart`, `CartReorder`, `Checkout`, `CheckoutRestApi`, `Sales` modules.
+
+
+{% info_block warningBox "Verification" %}
+
+ - Make sure that right after the Order amendment checkout the changes made in the order are not applied to the order.
+ - Make sure the Quote is saved to the `spy_sales_order_amendment_quote` table.
+ - Make sure the order items are in the `order amendment draft pending` state.
+ - Execute `console oms:check-condition` command, make sure the order items are in the `order amendment draft applied` state.
+ - Make sure the changes made in the order are applied to the order.
+ - Execute `console oms:check-condition` command, make sure the mail notification is sent to the customer that the order amendment has been applied successfully or has been failed.
```

Comment:
Why does running this command result in sending an email?

Reply:
I don't understand what should be adjusted:
- `Make sure that right after the Order amendment checkout the changes made in the order are not applied to the order.` - here we check the changes are not applied after Checkout is finished
- `Execute console oms:check-condition command` ... - here we are executing the console command
- `Make sure the changes made in the order are applied to the order.` here we check the changes are applied after the console command execution

Reply:
Will be clearer if I remove the `right` word?

Reply:
Will the output of `console oms:check-condition` show me that the email has been sent?